doc/source/serve/production.md

```diff
@@ -81,6 +85,23 @@ deployments:
 
 The `import_path` is the deployment graph's import path. When you deploy your config file, Serve will import your deployment graph using this path. Similarly, the `runtime_env` is the deployment graph's runtime environment. Serve will import the deployment graph inside this environment.
 
+`host` and `port` are HTTP options. They determine the host IP address and the port for your Serve application's HTTP proxies. These are optional settings and can be omitted. By default, the `host` will be set to `0.0.0.0` to expose your deployments publicly, and the port will be set to `8000`. If you're using Kubernetes, check out [the documentation on deploying Serve on Kubernetes](deploying-serve-on-kubernetes) to see how these parameters affect your setup.
+
+:::{warning}
+Once your Serve application is running, its `host` and `port` cannot be changed. Running `serve deploy` on a config file with different `host` and `port` options will fail, without making any changes to the running Serve application.
+
+The following CLI commands automatically start a Serve application on your Ray cluster if one isn't already running:
+
+* `serve deploy`
+* `serve config`
+* `serve status`
+* `serve delete` (however, this command will also delete the application)
+
+All these commands, except `serve deploy`, will start Serve with default `host` and `port` options. If you want non-default `host` and `port` options, make sure to first deploy a config with these options specified using `serve deploy`.
```

Comment:
I would suggest to omit this given it'll be fixed in the follow-up

Reply:
Sounds good, I omitted it.